packages/docs/utils/all-components-list.ts

```diff
@@ -31,7 +33,7 @@ export const allComponentsList = [
   'text',
   'textarea',
   'time-input',
-  'toast-stack',
+  'toast',
```

Comment:
You need to change the `/public/assets/all-toast-stack.png` filename to `toast.png` otherwise it will break the UI

Reply:
Updated.